packages/data-explorer-ui/src/components/common/Button/button.stories.tsx

```diff
@@ -23,6 +28,12 @@ const SecondaryButtonTemplate: ComponentStory<typeof Button> = (args) => (
   <ButtonSecondary {...args} />
 );
 
+const TextButtonWhiteTemplate: ComponentStory<typeof Button> = (args) => (
+  <Box sx={{ backgroundColor: "#333", borderRadius: 1, px: 10, py: 5 }}>
```

Comment:
Let's remove the styles around the button template - just in case we want to use WhiteTextButtonStory elsewhere in another story.

Reply:
Cleaned this up